### PR TITLE
feat: Change the screenshare capture fps from UI.

### PR DESCRIPTION
**Pull request by jallamsetty1**
Add the ability to change the capture frame rate for screenshare from the UI. The fps becomes effective only on screen shares that are started after the setting is changed.

Screenshots attached.
Screen Shot 2021-06-23 at 5 34 06 PM

Screen Shot 2021-06-23 at 5 33 56 PM

### CODE REVIEW

File: react/features/settings/constants.js [hunk 1]
```diff
@@ -9,3 +9,7 @@ export const SETTINGS_TABS = {
  * View ID for the Settings modal.
  */
 export const SETTINGS_VIEW_ID = 'SETTINGS_VIEW_ID';
+
+export const SS_DEFAULT_FRAME_RATE = 5;
+
+export const SS_SUPPORTED_FRAMERATES = [ 5, 15, 30 ];
```
Comment (damencho) [inline react/features/settings/constants.js:15]: And here.

File: react/features/screen-share/actionTypes.js [hunk 2]
```diff
@@ -10,3 +10,4 @@
  */
 export const SET_SCREEN_AUDIO_SHARE_STATE = 'SET_SCREEN_AUDIO_SHARE_STATE';
 
+export const SET_SCREENSHARE_CAPTURE_FRAME_RATE = 'SET_SCREENSHARE_CAPTURE_FRAME_RATE';
```
Comment (damencho) [inline react/features/screen-share/actionTypes.js:13]: Can you add jsdocs here, please.

File: react/features/settings/functions.js [hunk 3]
```diff
@@ -112,7 +115,9 @@ export function getMoreTabProps(stateful: Object | Function) {
             && isLocalParticipantModerator(state));
 
     return {
+        currentFramerate: `${framerate} frames-per-second`,
```
Comment (damencho) [inline react/features/settings/functions.js:118]: Can we do i18n for `frames-per-second` or at least change it to fps maybe ...?

**Review by damencho — CHANGES_REQUESTED** [review]

File: react/features/settings/functions.js (hunk 3 above)
Comment (jallamsetty1) [reply react/features/settings/functions.js:118]: I can translation for frames-per-second. This was in figma that I got from Tibby, so I believe product wanted to keep it that way.

**Review by damencho — APPROVED** [review]

**Pull request merged**